### PR TITLE
Use identifier instead of path

### PR DESCRIPTION
**Pull request by JoshuaBrest**
This is better beacuse i don't want my de instance to interfere with my real instance.

### CODE REVIEW

**Comment by JoshuaBrest** [comment]
Please note, user will have to reinstall the GPT but their whisky packages should remain.

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

File: Whisky/Utils/WineInstaller.swift [hunk 1]
```diff
@@ -13,26 +13,27 @@ class WineInstaller {
         .appendingPathExtension("zip")
 
     static let libraryFolder = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask)[0]
-        .appendingPathComponent("Whisky")
+        .appendingPathComponent(Bundle.main.bundleIdentifier ?? "com.isaacmarovitz.Whisky")
         .appendingPathComponent("Libraries")
 
     static func isWineInstalled() -> Bool {
-        return FileManager.default.fileExists(atPath: libraryFolder.path)
+        return FileManager.default.fileExists(atPath: libraryFolder.path())
```
[the remark this answers is not in the document]
Comment (JoshuaBrest) [reply Whisky/Utils/WineInstaller.swift:20]: The .path is deprecated

**Comment by JoshuaBrest** [comment]
@IsaacMarovitz Why was this closed?

**Comment by JoshuaBrest** [comment]
I will redo this with modern code.

**Pull request closed without merging**